HomeworkTwelve/src/main/java/Calculator/CallCalculator.java

```diff
@@ -0,0 +1,52 @@
+package Calculator;
+import java.util.Scanner;
+
+public class CallCalculator {
+
+    public static void main(String[] args) {
```

Comment:
you don't need this class in current homework, because you need to test CalculatorLogic methods